app/controllers/wizard/steps/import_dates_controller.rb

```diff
@@ -0,0 +1,32 @@
+module Wizard
+  module Steps
+    class ImportDatesController < BaseController
+      def index
```

Comment:
So effectively what we're doing here is rendering an individual question answer, right? Any reason we don't use `show` ?

Reply:
Because it's a static page it makes more sense to be rendered on the index. For show usually rails expects something like: `/resource/:id`. I know you are thinking about the `:commodity` in the url but `duty-calculator/:commodity/import_date` is not really a show. A show would be something like `duty-calculator/:commodity/import_date/:id`